### PR TITLE
Preview custom email

### PR DESCRIPTION
**Pull request by ivan-unfolds**

### CODE REVIEW

**Comment by rjmk** [comment]
Hi, @ivanmauricio. Looks like there are some problems with the flow tests

**Comment by rjmk** [comment]
Does this PR remove the ability to send emails?

**Comment by ivan-unfolds** [comment]
No, although I did change the error we respond with when sending an email so that we can handle it on the front end.

File: api/services/email_mailgun.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ module.exports = {
     var sendEmail = email => cb => {
       var address = email.to
       mailgun.messages().send(email, (error, results) =>
-        error ? cb({error: JSON.stringify(error), address}, null) : cb(null, {results: JSON.stringify(results), address})
+        error ? cb({ address }, null) : cb(null, { results: JSON.stringify(results), address })
```
Comment (rjmk) [inline api/services/email_mailgun.js]: I would add `message: error.message` to the error object

File: src/admin/dumb_components/custom_email_form.js [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
-      <form onSubmit={onSubmit}>
-        <input type='text' className='custom-email-subject' placeholder='Subject'></input>
-        <textarea className='custom-email' placeholder='Email body' />
-        <button type='submit'>Submit</button>
-      </form>
-    </div>
-  )
+  return mode === 'PREVIEW_CUSTOM'
+    ? preview_custom_email({ members, preview, props })
+    : <div className='custom-email-container'>
```
Comment (rjmk) [inline src/admin/dumb_components/custom_email_form.js]: I think this is too large a component to be inside a ternary. Maybe extract it out?

**Comment by rjmk** [comment]
Looks good apart from comments

**Pull request merged**